wiki/Submission/Beatmap_Ranking_Procedure/en.md

```diff
@@ -1,44 +1,23 @@
 
-Qualified / Ranked ![Heart icon](/wiki/shared/icon/heart.gif "Heart icon")
--------------------------------------------------------------------------
-
-Once a map is bubbled, a BN can qualify it if the map meets the two criteria listed above. The reason two approval team members are required to check a map is because human beings make mistakes, and this greatly reduces the times of that a map makes it through the ranking system with a drastic error. If a map gets qualified but approval team members or anyone else found it missed a drastic error (e.g. A 5mb video is included in the `.osz` that is not used), then it might get disqualified. After a map got into the qualified section for a certain amount of time (usually about one week) with no errors been found, it will automatically get ranked afterwards.
-
-Disqualified ![Heart-broken icon](/wiki/shared/icon/broken-heart.gif "Heart-broken icon")
-----------------------------------------------------------------------------------
-
-People make mistakes. Therefore, if a map makes it through the approval system with a drastic error, then it might become temporarily disqualified. If your map gets disqualified, you need to address the unrankable issues in order to fix what was wrong, so that your map may become re-qualified and be enjoyed as quickly as possible.
-
-Note: You'll sometimes be asked to change things you do not want to change. Sometimes you may be able to get away without changing it, but if half the Beatmap Nomination Group agree that you should change it, then let go of your pride and just fix it.
-
-Approved ![Fire icon](/wiki/shared/icon/flame.gif "Fire icon")
-------------------------------------------------------------
-
-Following up from [ranked](#qualified---ranked), a BN can approve these beatmaps if they are longer than 5 minutes long (aka [marathon](/wiki/Glossary/#marathon) beatmaps) and/or that do not follow the [ranking criteria](/wiki/Ranking_Criteria) but are considered "okay" by a BN.
-
-Note: Beatmaps (that are less than 5 minutes) are planned to be ranked, not approved!
+Once a map is in the Qualified section for around 7 days, it will be automatically moved to the Ranked section. If a map is disqualified, the 7 day timer is not reset, meaning if a map is disqualified after 4 days, once it is qualified again it will need 3 days before being moved to ranked. After the map has been moved to the Ranking Section, it will be there permanently and the map will award [Performance Points](/wiki/Performance_Points).
```

Comment:
```suggestion
Once a map is in the Qualified category for around 7 days, it will be automatically moved to the Ranked category. If a map is disqualified, the 7 day timer is not reset, meaning if a map is disqualified after 4 days, once it is qualified again it will need 3 days before being moved to ranked. After the map has been moved to the Ranking category, it will be there permanently and the map will award [Performance Points](/wiki/Performance_Points).
```